packages/gaudimarlinwrapper/package.py

```diff
@@ -33,5 +35,13 @@ def setup_run_environment(self, spack_env):
         spack_env.prepend_path('PYTHONPATH', self.prefix.python)
         spack_env.prepend_path("PATH", self.prefix.scripts)
 
+    @run_after('install')
+    def install_example(self):
+        install_tree('runit', os.path.join(self.prefix, 'runit'))
```

Comment:
@andresailer @fdplacide `runit` is a bit non-standard  and won't be in the path. Can I also add this to` scripts`?

Reply:
we should maybe call this `example`

Reply:
yes I agree, `example` would be a better fit. I will also change it in the GMP repo.
I also agree with moving this to scripts, and removing the `runit/` folder.

Reply:
ok, sounds good. And maybe install it to `share/GMPWrapper/example` in the CMake?

Reply:
yes, share is fine.